broker/lib/fabrik/DirectorInstance.js

```diff
@@ -321,18 +322,28 @@ class DirectorInstance extends BaseInstance {
         this.cloudController.getServiceInstance(this.guid),
         this.initialize(operation).then(() => this.manager.getDeploymentInfo(this.deploymentName))
       ])
-      .spread((instance, deploymentInfo) => ({
-        title: `${this.plan.service.metadata.displayName || 'Service'} Dashboard`,
-        plan: this.plan,
-        service: this.plan.service,
-        instance: _.set(instance, 'task', deploymentInfo),
-        files: [{
-          id: 'status',
-          title: 'Status',
-          language: 'yaml',
-          content: yaml.dump(deploymentInfo)
-        }]
-      }));
+      .spread((instance, deploymentInfo) => {
+        return Promise.all([
+          instance,
```

Comment:
Using a Promise.all against an already resolved variable looks a bit odd. Can this be optimized a bit further? Essentially, here, we are depending on the corollary that a constant is a resolved promise.
Can we not club the plan fetch operation with the instance fetch promise above? 
Something like 
```
this.cloudController.getServiceInstance(this.guid).then(instanceInfo => {
  this.cloudController.getServicePlan(instanceInfo.entity.service_plan_guid, {})
})
```

Of course, you need to preserve `instanceInfo` as well